test/test.js

```diff
@@ -619,6 +619,14 @@ describe('Page', function() {
       expect(error.message).toContain('Navigation Timeout Exceeded: 1ms');
       process.removeListener('unhandledRejection', unhandledRejectionHandler);
     }));
+    it('should disable timeout when its set to 0', SX(async function() {
+      server.setRoute('/empty.html', (req, res) => {});
+      let error = null;
+      page.goto(PREFIX + '/empty.html', {timeout: 0}).catch(e => error = e);
+      // const sleep = e => new Promise(ff => setTimeout(ff, e));
+      // await sleep(90000);
+      expect(error).toBe(null);
+    }));// 100000
```

Comment:
nit: please drop the comment here as well

Reply:
ok sir